charts/replicated-library/templates/lib/_statefulset.tpl

```diff
@@ -0,0 +1,72 @@
+{{/*
+This template serves as the blueprint for the StatefulSet objects that are created
+within the common library.
```

Comment:
"common library" should be `replicated library`

Reply:
resolved